.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "librdkafka"]
```

Comment:
I don't think so, but I can do some testing to make sure.

Reply:
I'm running into some problems with compilation that I've been troubleshooting. It seems the PPM can't find librdkafka anymore. The error occurs both with and without librdkafka added as a submodule, though, so it can be assumed that we don't need librdkafka as a submodule. This file has been removed.